docs/sources/alerting/alerting-rules/templates/_index.md

```diff
@@ -71,6 +71,11 @@ refs:
       destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/manage-notification-templates/#preview-notification-templates
     - pattern: /docs/grafana-cloud/
       destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/manage-notification-templates/#preview-notification-templates
+  reference-labels:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/latest/alerting/fundamentals/alert-rules/annotation-label/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/latest/alerting/fundamentals/alert-rules/annotation-label/
```

Comment:
I don't think https://grafana.com/docs/grafana-cloud/alerting-and-irm/latest/alerting/fundamentals/alert-rules/annotation-label/ exists

Reply:
Oh it's got a version in that it shouldn't:

```suggestion
      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/alert-rules/annotation-label/
```